cypress-io/cypress
# feat: Add status icon to Sidebar Runs Page

- Closes https://github.com/cypress-io/cypress/issues/27206

### Additional details
- Add visualization of `latest` run status for the Runs Icon

### Steps to test

- Launch Cypress with associated cloud run
- Verify `latest` run status matches icon

### How has the user experience changed?
Failing |  In-Errored | Cancelled | Passed | Failed | Running | No Activity
:-:|:-:|:-:|:-:|:-:|:-:|:-:
![Screen Shot 2023-08-25 at 1 35 08 PM](https://github.com/cypress-io/cypress/assets/22619500/ecb8a79b-f06a-4d42-9a9f-788342bb1f7c) | ![Screen Shot 2023-08-25 at 1 34 36 PM](https://github.com/cypress-io/cypress/assets/22619500/d6748e00-e089-4a0b-92ce-bfcda19dbf74) | ![Screen Shot 2023-08-25 at 1 34 25 PM](https://github.com/cypress-io/cypress/assets/22619500/26d7c154-ae64-4d35-a042-eda49ae51654) | ![Screen Shot 2023-08-25 at 1 34 15 PM](https://github.com/cypress-io/cypress/assets/22619500/d05798d2-3f72-4ed8-8718-96a3ee5ee50d) | ![Screen Shot 2023-08-25 at 1 34 03 PM](https://github.com/cypress-io/cypress/assets/22619500/75fac9f4-0dbc-468d-861f-a0aaf77038b4) | ![Screen Shot 2023-08-25 at 1 40 47 PM](https://github.com/cypress-io/cypress/assets/22619500/cc3fc2a3-a936-4701-909a-e67a71405241) | ![Screen Shot 2023-08-25 at 1 43 30 PM](https://github.com/cypress-io/cypress/assets/22619500/cfd48db4-8ab0-4a81-9f33-02c9595c4afc)

### PR Tasks


- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
